aten/src/ATen/Context.cpp

```diff
@@ -76,7 +76,9 @@ void check_fp32_prec_backend_and_op(
 
   C10_ALWAYS_INLINE void warn_deprecated_fp32_precision_api(){
     TORCH_WARN_ONCE(
-      "This API is going to be deprecated, please see "
+      "Please use the new API settings to control TF32 behavior, such as torch.backends.cudnn.conv.fp32_precision = tf32 "
```

Comment:
I don't recall if we have a way, from here, to set the warning stacklevel argument.
Setting that value properly will make it point to the user code setting the value and will be more obvious on what they did wrong.